### ISSUE TITLE
Request: Family Filter (opt youtube restricted mode) variant

### ISSUE THREAD

**Issue by Tomatoide** [AdguardTeam/AdGuardDNS#21]
While it is useful for kids, it can be a pain to use for an environment of adults with lots of false positives on youtube, so can we have a variant without forced youtube restricted mode?
Or if not possible is there a way to bypass it while still using family filter? thanks.

**Comment by ameshkov** [comment]
Well, but the family protection DNS is supposed to be used by kids.

To bypass it you'll need to add www.youtube.com with its normal address to your local hosts file

**Comment by Tomatoide** [comment]
Well it's setup on a home router that is used by both kids and adults, so this is where the problem comes from...
about the bypass method, could you please elaborate? what do you mean by youtube's normal address? thanks

**Comment by Tomatoide** [comment]
Ok, but is it possible to have a variant without forcing restricted mode? As I stated before we would love to have adblock and adult site blocking in a simple to configure dns but at the same time still be able to use youtube.. 
Restricted mode in youtube as it stands rn is a mess, it has a lot of false positives, while that can be a non-issue if the users were kids, it can be an annoying experience for adults.. since forcing it in family filter I can't use it on home router nor recommend it to people due to recurring complaints of youtube being unusable, were some troubleshooting videos for example weren't accessible, same with some tutorials.. so as it stands out family filter is not always used in kids only environments.. I think I have seen similar complaints also on the forum and reddit, so I hope we get a varient of family filter without forced restricted mode if possible.. thanks

**Comment by ameshkov** [comment]
@Tomatoide I understand, but that'd be too complicated to have a different DNS configuration for every user. That's why we have [AdGuard Home](https://github.com/AdguardTeam/AdguardHome) -- so that power users could install it and setup as they wish.

**Comment by Tomatoide** [comment]
Fair enough, I already knew about adguard home but it sounded like an overkill for my use case so I hoped a modified family filter would be a possibility, anyways thanks for responding.

**Comment by Tomatoide** [comment]
Update: so I decided to give adguard home a quick try, and unfortunately it also doesn't give you the option to disable forced restricted mode on youtube unless you disable safe search ENTIRELY on other search engines for example along with it, so sadly no solution in adguard home either...

**Comment by ameshkov** [comment]
@Tomatoide uh, my bad! The difference is that we will actually make it configurable there.

Meanwhile, the temporary solution would be to add `@@||youtube.com` to your custom filtering rule.

**Comment by Tomatoide** [comment]
@ameshkov The thing is, in my case, I don't have a permenant device to set adguard home on and leave it working 24/7, it's a simple home router and also I'm the "family tech guy" and I'm not here all the time so a simple dns setup on the home router with less probability for anything to go wrong is a much much better solution in this scenario..

Actually this is a suggestion for adguard dns I hope it can happen in the near future (should I open another issue for it? or may be rename this one?):

**_Default_**: 

- Block ads, trackers, malicious and phishing domains

**_Adult filter_**: 

- Block ads, trackers, malicious and phishing domains
- Block access to adult, pornographic and explicit sites, nsfw reddit subs
- Block proxies or VPNs 
- Enforce safe search on search engines

**_Family protection_**: 
- Block ads, trackers, malicious and phishing domains
- Block access to adult, pornographic and explicit sites
- Mixed content sites (like Reddit)
- Block proxies or VPNs 
- Enforce safe search on search engines
- Youtube restricted mode

**Comment by Archive5** [comment]
> Update: so I decided to give adguard home a quick try, and unfortunately it also doesn't give you the option to disable forced restricted mode on youtube unless you disable safe search ENTIRELY on other search engines for example along with it, so sadly no solution in adguard home either...

https://github.com/AdguardTeam/AdGuardHome/issues/1163

**Comment by Krakken77** [comment]
+1 for this feature request as described by Tomatoide, here above.

**Comment by dcreed** [comment]
I also agree with this request. I'm willing to pay a subscription cost as well.

**Comment by stbeep1** [comment]
I also agree with this request.

**Comment by GulnazSalakhova** [comment]
This request is now being processed in the current AdGuard DNS interface. Parental control provides separate options to block adult websites, secure search, and restrict access to YouTube. This allows users to block adult content and ensure safe search while keeping YouTube access restriction mode disabled.

Website categories can also be configured separately. Although this is implemented as independent settings and not as a separate preset “Adult Filter”, does this functionality satisfy the original request?

**Comment by GulnazSalakhova** [comment]
This request is now supported in AdGuard DNS private profiles.

Under Parental control, the following options can be configured independently:

- Block adult websites

- Safe search

- YouTube restricted mode

This allows users to enable adult-content blocking and safe search while leaving YouTube restricted mode disabled. Therefore, the original use case is covered without requiring a separate “Adult Filter” preset.

Since the requested functionality is already available, this issue can be closed.
